src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie,
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
```

Comment:
Better to make destructive assignment here

Reply:
I agree with @Dikey945